Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'sass-rails'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 
+gem 'webpack-rails'
```

Comment:
I have read the source of both.  they actually achieved the purpose using similar approach.

😄  to be short, webpack-rails is more general and more flexible IMO, it supports all types of assets (javascript, css, etc), and commands of webpack and rails are independent. 

webpacker only supports javascript. And you are supposed run `rails webpacker:compile` to get the assets digests, this implementation couples webpack with rails and does not allow development mode to have digests( although this is a minor issue ). It also designed for rails 5.1, I am not sure if it fits rails 4.2 yet.